src/managing-workflow/platform-monitoring.md

```diff
@@ -70,30 +71,14 @@ If you wish to have persistence for Grafana you can set `enabled` to `true` in t
      size: 5Gi # PVC size
 ```
 
-You have to set (if you do not have it already) `standard` StorageClass as per [PVC Dynamic Provisioning](#pvc-dynamic-provisioning), as it does not get set by default in Kubernetes v1.4.x and v1.5.x.
-
-
 ### Off Cluster Grafana
 
 If you wish to provide your own Grafana instance you can set `grafana_location` in the `values.yaml` file before running `helm install`.
 
-## InfluxDB
-
-InfluxDB writes data to the host disk, however, if the InfluxDB pod dies and comes back on
-another host, the data will not be recovered you need to enable on-cluster persistence for data to persist. The InfluxDB Admin UI is also
-exposed through the router allowing users to access the query engine by going to `influx.mydomain.com`. You will need to
-configure where to find the `influx-api` endpoint by clicking the "gear" icon at the top right and changing the host to
-`influxapi.mydomain.com` and port to `80`.
-
-** Note: Each user accessing the Influx UI will need to make this change. **
-
-You can choose to not expose the Influx UI and API to the world by updating
-`$CHART_HOME/workspace/workflow-$WORKFLOW_RELEASE/manifests/deis-monitor-influxdb-api-svc.yaml` and
-`$CHART_HOME/workspace/workflow-$WORKFLOW_RELEASE/manifests/deis-monitor-influxdb-ui-svc.yaml` and removing the
-following line - `router.deis.io/routable: "true"`.
+## [InfluxDB](https://docs.influxdata.com/influxdb)
+InfluxDB writes data to the host disk, however, if the InfluxDB pod dies and comes back on another host, the data will not be recovered you need to enable on-cluster persistence for data to persist. The InfluxDB Admin UI is also exposed through the router allowing users to access the query engine by going to `influx.mydomain.com`. You will need to configure where to find the `influx-api` endpoint by clicking the "gear" icon at the top right and changing the host to `influxapi.mydomain.com` and port to `80`.
```

Comment:
`...the data will not be recovered you need to enable on-cluster persistence...` needs attention